packages/time-picker/src/vaadin-time-picker-mixin.js

```diff
@@ -202,14 +167,53 @@ export const TimePickerMixin = (superClass) =>
       return this.$.clearButton;
     }
 
+    /**
+     * The object used to localize this component. To change the default
+     * localization, replace this with an object that provides all properties, or
+     * just the individual properties you want to change.
+     *
+     * The object has the following JSON structure:
+     *
+     * ```
+     * {
+     *   // A function to format given `Object` as
+     *   // time string. Object is in the format `{ hours: ..., minutes: ..., seconds: ..., milliseconds: ... }`
+     *   formatTime: (time) => {
+     *     // returns a string representation of the given
+     *     // object in `hh` / 'hh:mm' / 'hh:mm:ss' / 'hh:mm:ss.fff' - formats
+     *   },
+     *
+     *   // A function to parse the given text to an `Object` in the format
+     *   // `{ hours: ..., minutes: ..., seconds: ..., milliseconds: ... }`.
+     *   // Must properly parse (at least) text
+     *   // formatted by `formatTime`.
+     *   parseTime: text => {
+     *     // Parses a string in object/string that can be formatted by`formatTime`.
+     *   }
+     * }
+     * ```
+     *
+     * Both `formatTime` and `parseTime` need to be implemented
+     * to ensure the component works properly.
```

Comment:
Think this statement can be removed with the partial i18n approach, right?

Reply:
That part is indeed a bit confusing at the moment. Technically you can omit either function, but then the component likely won't work correctly, as the component logic depends on the parse function to work with what the format function produces. But I didn't have any good ideas on how to make that clearer. I can take another look if I can formulate this better.

Reply:
I see. We can keep probably keep it, and consider this change a refactoring.